components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/mgt/impl/PolicyManagerImpl.java

```diff
@@ -726,4 +726,17 @@ public boolean setPolicyApplied(DeviceIdentifier deviceIdentifier) throws Policy
             throw new PolicyManagementException(msg, e);
         }
     }
+
+    @Override
+    public int getPolicyCount() throws PolicyManagementException {
+        int policyCount = 0;
+        try {
+            policyCount = policyDAO.getPolicyCount();
+            return policyCount;
+        } catch (PolicyManagerDAOException e) {
+            String msg = "Error occurred while getting policy count";
+            log.error(msg, e);
```

Comment:
Fixed in commit [1]
[1] https://github.com/wso2/carbon-device-mgt/commit/d7e363cef74f7700585f893253f5b0380fa26e40